### PR TITLE
Michaelkatsoulis/cer 1370 deploy deepfloyd text image model

### PR DESCRIPTION
**Pull request by Katsie011**

### CODE REVIEW

File: 11-deepfloyd-text-image-model/main.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    prompt : str
+    # image:  Optional[]
+    # url:  Optional[str]
+
+
+########################################################################
+# Run once on first deployment to initialise the model
+########################################################################
+
+access_token = '<<<<Your Access Token Here>>>>'
```
Comment (jonoirwinrsa) [inline 11-deepfloyd-text-image-model/main.py:24]: @Katsie011 could be more specific. I'm worried users will think its their cerebrium api key

File: 11-deepfloyd-text-image-model/main.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+# Run every time your model is called.
+########################################################################
+
+def predict(item, run_id, logger):
+    item = Item(**item)
+    ##Do something with parameters from item
+    # Retrieve text embeddings
+    prompt_embeds, negative_embeds = stage_1.encode_prompt(Item.prompt)
+
+    # stages = []
```
Comment (jonoirwinrsa) [inline 11-deepfloyd-text-image-model/main.py:55]: @Katsie011 do we want the commented out code? Might confuse users

File: 11-deepfloyd-text-image-model/main.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    logger.info(f'Starting with prompt: {Item.prompt}')
+
+    # Running stage 1
+    generator = torch.manual_seed(0)
+    image = stage_1(prompt_embeds=prompt_embeds, negative_prompt_embeds=negative_embeds,
+                     generator=generator, output_type="pt").images
+    # stages.append(pt_to_pil(image)[0])
+    logger.info('Run stage 1')
```
Comment (jonoirwinrsa) [inline 11-deepfloyd-text-image-model/main.py:63]: minor but comment and code order seem wrong, or wrong wording.

Perhaps swap order or reword? `logger.info('Stage 1 complete')`

**Comment by Katsie011** [comment]
Thanks for catching those Jono!

**Pull request merged**